### PR TITLE
Srushti gangireddy/populate source count

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Run_achilles_queries.sh changed to populate source counts.
2. make_bq_data.sh changed to populate counts in concept table.

### CODE REVIEW

**Comment by biopete** [comment]
Also, we need to add a new achilles_results.csv and concept.csv with the source_count_value field.

**Comment by SrushtiGangireddy** [comment]
Will update the api calls with the source count value.

File: api/db-cdr/generate-cdr/make-bq-public-data.sh [hunk 1]
```diff
@@ -131,11 +128,23 @@ set count_value =
...
+        when count_value  > 0 and count_value >= ${BIN_SIZE}
+            then ROUND(ROUND(count_value / ${BIN_SIZE}) * ${BIN_SIZE}/ ${person_count}, 2)
+        when source_count_value  > 0 and source_count_value < ${BIN_SIZE}
+            then ROUND(${BIN_SIZE} / ${person_count},2)
+        when source_count_value  > 0 and source_count_value >= ${BIN_SIZE}
+            then ROUND(ROUND(source_count_value / ${BIN_SIZE}) * ${BIN_SIZE}/ ${person_count}, 2)
+        else
+            0.00
     end
 where count_value > 0"
```
Comment (biopete) [inline api/db-cdr/generate-cdr/make-bq-public-data.sh]: We can't do where count_value > 0 here. We have to set all to 20 when it's 0.

File: api/db-cdr/generate-cdr/generate-cdr-counts.sh [hunk 2]
```diff
@@ -86,15 +86,6 @@ PUBLIC_DATASET=public$CDR_VERSION
 startDate=`date`
 echo `date` " Starting generate-cdr-counts $startDate"
 
-## Make BigQuery denormalized tables
```
Comment (biopete) [inline api/db-cdr/generate-cdr/generate-cdr-counts.sh]: This code needs to be put back in

Comment (SrushtiGangireddy) [reply api/db-cdr/generate-cdr/generate-cdr-counts.sh]: Put it back.

**Comment by SrushtiGangireddy** [comment]
Added the csvs.

**Comment by biopete** [comment]
I approve once these tests pass

**Pull request merged**